.github/stale.yml

```diff
@@ -0,0 +1,55 @@
+# Configuration for probot-stale - https://github.com/probot/stale
+
+# Number of days of inactivity before an Issue or Pull Request becomes stale
+daysUntilStale: 730
+
+# Number of days of inactivity before an Issue or Pull Request with the stale label is closed.
+# Set to false to disable. If disabled, issues still need to be closed manually, but will remain marked as stale.
+daysUntilClose: 14
+
+# Issues or Pull Requests with these labels will never be considered stale. Set to `[]` to disable
+exemptLabels:
+  - pinned
+  - security
+  - "[Status] Maybe Later"
+  - "priority/high"
+
+# Set to true to ignore issues in a project (defaults to false)
+exemptProjects: false
```

Comment:
IMO this should be `true`.

Reply:
Agreed, though it probably shouldn't make a difference in our repo - if I understand properly, adding something to a project would count as activity, so for this to come up with our other settings, the issue would need to be added to a project that's still open 2 years later. Shifting the unfixed issue from a project that's being closed into a project for the next release should count as activity.